src/components/Widget/inputs.tsx

```diff
@@ -96,9 +104,10 @@ export function useSyncWidgetInputs({
     () => ({
       type,
       amount,
-      ...tokens,
+      // If the default has not yet been handled, preemptively disable the widget by passing no tokens.
```

Comment:
Add one more comment:

    Effectively, this resets the widget - avoiding rendering stale state - because with no tokens the skeleton will be rendered.